### PR TITLE
Fixed behat typos

### PR DESCRIPTION
**Pull request by wachterjohannes**
Main PR: https://github.com/sulu-io/sulu-docs/pull/26

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/Behat/AdminContext.php [hunk 1]
```diff
@@ -365,6 +365,7 @@ public function thereShouldBeErrors($expectedErrorCount)
 
     /**
      * @Given I expect a data-navigation to appear
+     * @Given I expect wait for data-navigation to appear
```
Comment (dantleech) [inline src/Sulu/Bundle/AdminBundle/Behat/AdminContext.php]: `I wait for a data-navigation to appear`

**Comment by wachterjohannes** [comment]
@dantleech fixed

File: src/Sulu/Bundle/AdminBundle/Behat/AdminContext.php [hunk 2]
```diff
@@ -365,6 +365,7 @@ public function thereShouldBeErrors($expectedErrorCount)
 
     /**
      * @Given I expect a data-navigation to appear
+     * @Given I expect wait for a data-navigation to appear
```
Comment (dantleech) [inline src/Sulu/Bundle/AdminBundle/Behat/AdminContext.php]: `I wait for a given ..` no `expect`.

**Comment by wachterjohannes** [comment]
@dantleech fixed

**Comment by dantleech** [comment]
doh. and it needs rebase.

**Comment by wachterjohannes** [comment]
@dantleech fixed

**Pull request merged**